Automattic/wp-calypso
# Jetpack Cloud: Fix failed backup state design

#### Changes proposed in this Pull Request

* Fix the backup failed state in `DailyBackupStatus` so that it more closely matches the Figmas

#### Testing instructions

* Check daily backup status for a date that has a failed backup. Force one manually if needed in `renderBackupStatus()`. Check to be sure that the design closely matches the Figma, and the new (proper) icon is present.

Fixes 1142395350490785-as-1173118334902057
